src/languages/es.js

```diff
@@ -148,6 +148,8 @@ export default {
     avatarCropModal: {
         title: 'Editar foto',
         description: 'Arrastra, haz zoom y rota tu imagen para que quede como te gusta.',
+        rotateImage: 'Girar',
```

Comment:
```suggestion
        rotateImage: 'Rotar',
```

Heard back from our people on this translation. Let's use "Rotar".

Reply:
Done